components/Feature/Services/index.js

```diff
@@ -99,135 +124,179 @@ const Services = ({
+                {!feedbackSubmitted ? (
+                  <Details id="feedback-summary" title="Provide feedback">
+                    <form id="search-feedback-form" onSubmit={sendFeedback}>
+                      <div className="govuk-inset-text">
+                        <div>
+                          <label htmlFor="search-feedback">Search results feedback</label>
+                        </div>
+                        <textarea
+                          id="search-feedback"
+                          name="search-feedback"
+                          maxLength="450"
+                          className={styles['feedback-textarea']}
+                        />
+                        <br />
+                        <button type="submit" className={`govuk-button`}>
+                          Send
+                        </button>
                       </div>
-                      <textarea
-                        id="search-feedback"
-                        name="search-feedback"
-                        maxLength="450"
-                        className={styles['feedback-textarea']}
+                    </form>
+                  </Details>
+                ) : (
+                  <div className={styles['success-message']}>
+                    Feedback submitted. Thank you for providing us with information to assist in
+                    improving this tool.
+                  </div>
+                )}
+              </div>
+
+              {filteredResources.resources.slice(0, 8).map(resource => (
+                <ResourceCard
+                  data-testid={`resource-${resource.id}`}
+                  resource={resource}
+                  referralCompletion={referralCompletion}
+                  setReferralCompletion={setReferralCompletion}
+                  detailsClicked={detailsClicked}
+                  openReferralForm={openReferralForm}
+                  referralData={referralData}
+                  setReferralData={setReferralData}
+                  referrerData={referrerData}
+                  setReferrerData={setReferrerData}
+                  updateSignpostSummary={updateSignpostSummary}
+                  signpostSummary={signpostSummary}
+                  setResidentInfo={setResidentInfo}
+                  token={token}
+                  referralSummary={referralSummary}
+                  setReferralSummary={setReferralSummary}
+                  updateEmailBody={updateEmailBody}
+                  setEmailBody={setEmailBody}
+                  residentInfo={residentInfo}
+                  setPreserveFormData={setPreserveFormData}
+                  preserveFormData={preserveFormData}
+                  wordsToHighlight={wordsToHighlight}
+                />
+              ))}
+
+              {filteredResources.resources.length > 8 && (
+                <>
+                  <div className={showMoreResults ? '' : styles.hidden} id="show-more-container">
```

Comment:
it's just a personal preference but I think I'd just conditionally render if showMoreResults (&&) instead of rendering it and just hiding from display (no need to change)

Reply:
This was over-engineered. Simplified!